libs/components/ag-grid/src/lib/modules/ag-grid/cell-editors/cell-editor-currency/cell-editor-currency.component.spec.ts

```diff
@@ -107,6 +105,8 @@ describe('SkyCellEditorCurrencyComponent', () => {
         rowHeight: 100,
       };
 
```

Comment:
AG Grid's implementation for `getActualWidth` and the column width change events no longer worked with a detached column the way we had been testing. Added mocks.